src/ft2font_wrapper.cpp

```diff
@@ -370,6 +372,24 @@ static int PyFT2Font_init(PyFT2Font *self, PyObject *args, PyObject *kwds)
     open_args.flags = FT_OPEN_STREAM;
     open_args.stream = &self->stream;
 
+    if (fallback_list && PyList_Check(fallback_list)) {
+        Py_ssize_t size = PyList_Size(fallback_list);
+
+        for (Py_ssize_t i = 0; i < size; ++i) {
+            // this returns a borrowed reference
+            PyObject* item = PyList_GetItem(fallback_list, i);
+            // Increase the ref count, we will undo this in dealloc
+            // this makes sure things do not get gc'd under us!
+            Py_INCREF(item);
+            self->fallbacks.push_back(item);
+            // TODO: check whether item is actually an FT2Font
```

Comment:
Just something like `PyObject_IsInstance(item, &PyFT2Font_Type)`, no?  Otherwise this will segfault if you pass in a non-ft2font?